src/model/FeaturesView.tsx

```diff
@@ -63,13 +66,18 @@ const columns = {
   },
   description: {
     label: 'Description',
-    resolver: (d: ProtvistaFeature) => d.description,
+    resolver: (d: ProtvistaFeature) => {
+      return `${d.description} ${d.evidences &&
+        d.evidences.map(evidence => `(${evidence.evidenceCode}) `)}`;
```

Comment:
Embedding too much inside of template strings makes it difficult read.

Reply:
This is temporary as we need a strategy for evidence tags to work both in website and in Nightingale, so using it as a placeholder.